### DISCUSSION TITLE
Mise en forme du texte

### DISCUSSION THREAD

**Discussion by JolyaneL** [MTESSDev/FRW#299] [Annonces]
Bonjour,

Je souhaite mette en forme le texte dans les libellés de questions. Par exemple, mettre de l'italic, un exposant etc.
![image](https://github.com/user-attachments/assets/01262ac0-5014-44e8-90a0-8cfd4d578783)

exemple de balise que j'ai besoin d'intégrer dans un libellé : 
`<em> ou <i>, <sup>, <sub>, <a>`

**Comment by j0ber** [comment]
Bonjour,

Il n'est pas possible d'insérer du HTML dans les libellés et dans les précisions. Ça ouvrirait la porte à trop de choses qui risqueraient de ne pas respecter les [règles du système de design du gouvernement du Québec](https://design.quebec.ca/design/composants/formulaire/champ-texte) .

Dans votre exemple, votre précision est beaucoup trop longue... Normalement une précision c'est court. Vous devriez plutôt utiliser une infobulle. Dans cette dernière vous aurez le loisir de mettre le html souhaité. Dans votre cas j'imagine que vous pourriez mettre un lien vers le guide en question? Ce serait encore mieux.

Voir https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=controlesBase (propriété tooltip). Il n'y a pas d'exemple avec du HTML mais vous pouvez en mettre.

Pour le "sup" dans le libellé inscrivez simplement m3 ou mètres cubes
